tests_react_native/example/notifications.ts

```diff
@@ -4,8 +4,8 @@ import {
   AndroidLaunchActivityFlag,
   AndroidCategory,
   AndroidImportance,
+  AndroidFlags,
```

Comment:
This fixes double loading of module which caused that `onBackgroundEvent` was not properly registered

Reply:
thank you!

Reply:
Yes, exactly.